### ISSUE TITLE
404 (Not Found) - nova-vendor/wdelfuego/nova4-customizable-footer/footer-content

### ISSUE THREAD

**Issue by evertonpessoa** [wdelfuego/nova4-customizable-footer#2]
In my project I am getting 404.
The solution I made was to generate the route using the code

use Illuminate\Support\Facades\Route;
Route::middleware(['nova'])
            ->prefix('nova-vendor/wdelfuego/nova4-customizable-footer')
            ->group(__DIR__.'/../routes.php');

**Comment by wdelfuego** [comment]
Hi Everton, thanks for reporting! That's strange behavior, I'd like to investigate.

A few questions:
- I assume you've added that route code to the `boot()` method in `vendor/wdelfuego/nova4-customizable-footer/src/ServiceProvider.php`, correct?
- If so, could you put the following code at the start of that method and check that the path that shows exists on your system:
```
   dd($this->app->basePath('vendor/wdelfuego/nova4-customizable-footer/routes.php'));
```

**Comment by davidemorotti** [comment]
Same issue here, the routes.php file exists, but not it's not registering :S

**Comment by wdelfuego** [comment]
Thanks for the info! I’ll look into this later today and release a fix asap.

**Comment by wdelfuego** [comment]
Fixed in v1.0.2, just released the package, please run `composer update`
